common/addressing/defaults.go

```diff
@@ -32,7 +32,7 @@ const (
 	// Default IPv4 prefix length of entire cluster
 	DefaultIPv4ClusterPrefixLen = 8
 	// Default IPv6 prefix to represent NATed IPv4 addresses
-	DefaultNAT46Prefix = "aa46::/48"
+	DefaultNAT46Prefix = "f00d::/48"
```

Comment:
To solve the conflict issue we can also dedicate another bit from the unused portion of the address or a special state -1 that will never be assigned. If we dedicate another bit, I think it will not address one of the goals of this change, i.e use a common prefix for ipv6 container addresses and nat46. If we use a special state we could also get this working between containers. Pl let me know your thoughts.